registry/storage/linkedblobstore.go

```diff
@@ -398,6 +399,16 @@ func (lbs *linkedBlobStatter) Clear(ctx context.Context, dgst digest.Digest) (er
 		return err
 	}
 
+	// See https://github.com/docker/distribution/issues/3322 to get more detail
```

Comment:
The comment should explain the code. It can also link to a work item if you like, but the explanation should be in here.